app/models/google_event.rb

```diff
@@ -111,6 +105,20 @@ def load_emails
       ConferenceRoom.pluck(:email)
     end
 
+    GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
+    def add_events_from_room(room, service, events, config)
+      service.list_events(room.email, config) do |result, _|
+        if result
+          result.items.each do |event|
+            next if event.attendees.find(&:self).response_status == GOOGLE_EVENT_DECLINED_RESPONSE
```

Comment:
I'd name the conditional. It also heavily violates Law of Demeter. Is `event` an object of class you implement or does it come from Google's implementation?